vocabularies/Org.OData.JSON.V1.xml

```diff
@@ -68,10 +132,57 @@
         <Annotation Term="Core.RequiresType" String="Edm.Stream" />
         <Annotation Term="Core.Description" String="The JSON Schema for JSON values of the annotated property, parameter, return type, term, or type definition" />
         <Annotation Term="Core.LongDescription">
-          <String>The schema can be a schema reference, i.e. `{"$ref":"url/of/schemafile#/path/to/schema/within/schemafile"}`</String>
+          <String>The schema can be a reference, i.e. `{"$ref":"url/of/schemafile#/path/to/schema/within/schemafile"}`</String>
         </Annotation>
       </Term>
 
+      <Function Name="query" IsBound="true" IsComposable="true">
+        <Annotation Term="Core.Description" String="Query a stream value of media type `application/json`, returning a stream value of media type `application/json`" />
+        <Annotation Term="Core.LongDescription">
+          <String>Extracts a JSON value, such as an array, object, or a JSON scalar value (string, number, boolean, or `null`) from the `input` JSON value:
+- If `path` only consists of the root selector followed by member and index selectors, it returns the identified single node within `input`, or `null` if no node is identified. 
+- If `path` potentially identifies multiple nodes within `input` (by using descendant, wildcard, union, array subset, or filter selectors), it returns an array containing the identified nodes, or an empty array if no node is identified. 
+- If `input` is not a valid JSON value, the function returns `null`.
+- If `path` is `null`, not a valid [JSONPath expression](#Path), or does not match the structure of `input` (for example applying an index selector to a scalar value), the function returns `null`. 
+          </String>
+        </Annotation>
+        <Parameter Name="input" Type="JSON.JSON">
+          <Annotation Term="Core.Description" String="JSON input" />
+        </Parameter>
+        <Parameter Name="path" Type="JSON.Path">
+          <Annotation Term="Core.Description" String="JSONPath expression to be applied to value of `expr`" />
+        </Parameter>
+        <ReturnType Type="JSON.JSON">
+          <Annotation Term="Core.Description" String="JSON value resulting from applying `path` to `input`" />
+        </ReturnType>
+      </Function>
+
+      <Function Name="value" IsBound="true" IsComposable="true">
```

Comment:
How can a function returning a primitive type be composable?

Reply:
Good catch!